hardhat.config.js

```diff
@@ -1,6 +1,6 @@
-require('dotenv').config();
```

Comment:
Again all changes should be reverted here besides the addition of the `blockNumber` used for tests